src/dataset/dataset_coco.py

```diff
@@ -242,17 +242,20 @@ def extract_patches(self, output_dir: str, patch_size: int, stride: int, min_are
 
         for image_data in tqdm(self.images):
             image_path = os.path.join(self.data_directory_path, image_data["file_name"])
-            
+
```

Comment:
Please remove these spaces/tabs. If you open the project in VSCode using the workspace file, these spaces should be removed automatically when saving file

![image](https://github.com/laradru/ai-palatal-rugoscopy/assets/26142253/cc7a5bb2-9ace-4311-9e9a-b7ba9c30d2b9)

Reply:
I only use the workspace, I don't how why these things are happening. Will keep an eye out, though. Thanks!